examples/ct/handlers_test.go

```diff
@@ -346,8 +357,8 @@ func TestAddChain(t *testing.T) {
 			if got, want := ct.Version(resp.SCTVersion), ct.V1; got != want {
 				t.Errorf("resp.SCTVersion=%v; want %v", got, want)
 			}
-			if got, want := hex.EncodeToString(resp.ID), ctMockLogID; got != want {
```

Comment:
In general, it's probably better to use `fmt.Errorf("%x", data)` when printing data for the screen.